README.md

```diff
@@ -5,9 +5,33 @@ The mbed OS driver for the ESP8266 WiFi module
 ESP8266 modules come in different shapes and forms, but most important difference is which firmware version it is programmed with. To make sure that your module has mbed-os compatible firmware follow update guide: https://developer.mbed.org/teams/ESP8266/wiki/Firmware-Update
 
 ## Testing
-The ESP8266 library contains the core network tests taken from mbed OS. After installing mbed CLI and importing the mbed OS library, the tests can be ran with the `mbed test` command:
+The ESP8266 library contains the core network tests taken from mbed OS. To run the tests you will need mbed CLI and mbed OS.
+
+First, setup the the esp8266-driver and mbed-os repositories for testing:
+``` bash
+# Sets up the ESP8266 for testing
+mbed clone esp8266-driver
+cd esp8266-driver
+mbed import mbed-os
+```
+
+Now you should be able to run the network tests with `mbed test`:
 ``` bash
 # Runs the ESP8266 network tests, requires a wifi access point
-mbed test -t <COMPILER HERE> -m <BOARD HERE> -n tests-net* --compile -DMBED_CFG_ESP8266_SSID='"<SSID HERE>"' -DMBED_CFG_ESP8266_PASS='"<PASS HERE>"'
+mbed test -t <COMPILER HERE> -m <BOARD HERE> -n tests-net* --compile -DMBED_CFG_ESP8266_SSID='<SSID HERE>' -DMBED_CFG_ESP8266_PASS='<PASS HERE>'
+mbed test -t <COMPILER HERE> -m <BOARD HERE> -n tests-net* --run --verbose
+```
```

Comment:
That's only if the esp8266 is imported into the mbed-os repo, in this case the mbed-os repo is imported into the esp8266